packages/wonder-blocks-modal/components/one-column-modal.md

```diff
@@ -56,8 +56,7 @@ const styles = StyleSheet.create({
 </View>;
 ```
 
-This example shows that `OneColumnModal` works with content and footer elements
-that aren't block level elements.
+This example shows a `OneColumnModal` with content and footer:
```

Comment:
This felt like a more useful example (I prefer to show examples of what we want people to do, rather than anti-patterns).

Reply:
It seems reasonable to pass a single button to the footer.

Reply:
See, that's what you did... you just phrased it differently.  sgtm.  🙂